src/main/java/thePackmaster/cards/intriguepack/Exultation.java

```diff
@@ -0,0 +1,53 @@
+package thePackmaster.cards.intriguepack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Exultation extends AbstractIntrigueCard {
+    public final static String ID = makeID("Exultation");
+
+    public Exultation() {
+        super(ID, 2, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        magicNumber = baseMagicNumber = 3;
+        baseBlock = 0;
```

Comment:
Hmm, this is a weird edgecase. If you do this you'll mess with some stuff that expects block cards to use their block instead of overwriting it. Might be better to leave baseBlock at -1 until needed since you overwrite it anyway.